### PR TITLE
DOCSP-43422 - Add Python 3.13

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-43422
Staging - https://preview-mongodbmongokart.gatsbyjs.io/pymongo/docsp-43422-python-version/compatibility/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/includes/language-compatibility-table-pymongo.rst [hunk 1]
```diff
@@ -91,6 +111,7 @@ Python 3
      -
 
    * - 3.13
```
Comment (mongoKart) [inline source/includes/language-compatibility-table-pymongo.rst:113]: Tech reviewer: This table says v3.13 of the driver supports CPython 3.11. Is that accurate?

**Review by mcmorisi — APPROVED** [review]
LGTM!

File: source/includes/language-compatibility-table-pymongo.rst (hunk 1 above)
Comment (blink1073) [reply source/includes/language-compatibility-table-pymongo.rst:113]: Yes, that is accurate

**Review by blink1073 — APPROVED** [review]
LGTM!

**Pull request merged**